### PR TITLE
[plugin.video.diy] 3.0.4

### PR DESCRIPTION
**Pull request by learningit**

### CODE REVIEW

File: plugin.video.diy/resources/lib/scraper.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     fanart = self.addonFanart
+     for i, (url, thumb, name, vidcnt) in list(enumerate(a[1:], start=1)):
+       name=name.strip()
+       vidcnt = vidcnt.strip()
+       html = self.getRequest(url.rsplit('/',1)[0])
+       plot = re.compile('"og:description" content="(.+?)"',re.DOTALL).search(html)
+       if plot is not None: plot = plot.group(1)
+       else: plot = name
+       infoList = {}
+       infoList['TVShowTitle'] = name
```
Comment (phil65) [inline plugin.video.diy/resources/lib/scraper.py]: TVShowTitle, but no mediatype?

File: plugin.video.diy/resources/lib/scraper.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+           url     = b['releaseUrl']
+           dirty = True
+           name    = h.unescape(b['title'])
+           fanart  = 'http://www.hgtv.com%s' % b['thumbnailUrl']
+           thumb   = 'http://www.hgtv.com%s' % b['thumbnailUrl']
+
+           infoList = {}
+           infoList['Duration']    = b['length']
+           infoList['Title']       = name
+           infoList['Studio']      = b['publisherId']
```
Comment (phil65) [inline plugin.video.diy/resources/lib/scraper.py]: also mediatype here I think?

File: plugin.video.diy/resources/lib/scraper.py (hunk 1 above)
Comment (learningit) [inline plugin.video.diy/resources/lib/scraper.py]: That's really odd, the diy add-on is derived from the hgtv add-on and it's in that one. I'll clean this up tomorrow. Thanks.

File: plugin.video.diy/resources/lib/scraper.py (hunk 2 above)
Comment (learningit) [inline plugin.video.diy/resources/lib/scraper.py]: Done.

**Pull request merged**